libs/api/ebpf_api.cpp

```diff
@@ -1397,6 +1397,14 @@ _clean_up_ebpf_object(_In_opt_ ebpf_object_t* object) noexcept
         clean_up_ebpf_programs(object->programs);
         clean_up_ebpf_maps(object->maps);
 
+        if (object->native_module_handle != ebpf_handle_invalid) {
+            // TODO: Investigate this.
```

Comment:
is there an issue number?

Reply:
I will remove this comment once PR #1548 is merged.

Reply:
Removed this.